### PR TITLE
Review for Newsweek website clone

### PR DESCRIPTION
**Pull request by chrystalme**
# Building a Responsive Website using Bootstrap and SASS. 

## Newsweek website clone using HTML, and Bootstrap framework. 

- HTML
- Bootstrap
- Responsive Design
- Mobile-first approach

### CODE REVIEW

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required


### Hi @team :wave: 


Great effort so far, but there are some updates to make.

- [x] On large screens, @media >= 998px, **Opinion** section overlap **My Turn** section.
- [x] In the **Culture & Travel** section, @media >= 1100px, the label, **Total Solar Eclipse**, is overlapping the images.
- [x] Resolve the linter errors.

That's it, make these changes, submit another form for a code review.

### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

File: README.md [hunk 1]
```diff
@@ -1,69 +1,44 @@
...
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the current project doesn't require testing.**
```
Comment (descholar-ceo) [inline README.md:33]: [Required] Remove this text. It is not needed, for users. It was put there to help you understand what you should write.

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @chrystalme and @sohaibya  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [ ] In `The debate` section the black VS text which is button should be centered as it is in the original, and that button should be passed by a horizontal line
- [ ] The subscribe section, the two overlapping images get distorted on mobile screens, check screens < 350 px

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-17 19-58-04 |Screenshot from 2020-12-17 20-00-48|

:point_right: Could you see the difference :question: :question: :question: 

- [ ] The license link from the readme file is broken, fix it
- [ ] This [@afam_chrys](https://twitter.com/afam_chrys) from the readme file doesn't exist on twitter, fix that as well
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

File: index.html [hunk 2]
```diff
@@ -1600,57 +1652,121 @@ <h5><a class="text-decoration-none text-dark " href="#">
...
-            <div class="section__subscribe">
-                <div class="section_newsletter-header my-3">
-                    Subscribe
+        <div class="d-lg-none">
+            <section class="section__newsletter">
+                <div class="section__subscribe">
+                    <div class="section_newsletter-header my-3">
+                        Subscribe
+                    </div>
+                    <div style="height: 400px;" class="jumbotron jumbotron-fluid d-flex flex-column flex-md-row
```
Comment (umairarshadbutt) [inline index.html]: Remove inline-style and move it to CSS file

File: index.html [hunk 3]
```diff
@@ -1600,57 +1652,121 @@ <h5><a class="text-decoration-none text-dark " href="#">
...
+                            <img src="images/newsletter-1.jpg" alt="...">
+                            <img class="news-letter-img" src="images/cover-mag-footer.jpg"
+                                alt="...">
+                        </div>
+
+                        <div class="d-flex flex-column align-items-center text-center mt-5 ml-md-5 mt-md-3">
+                            <span>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</span>
+                            <p class="mt-3">All Access + Weekly Delivery.
+                                Choose A Membership That's Perfect for You!</p>
+                            <button style="width: 40%;" class="btn btn-danger" type="button">Subscribe &gt;</button>
```
Comment (umairarshadbutt) [inline index.html]: Remove inline-style and move it to CSS file

File: index.html [hunk 4]
```diff
@@ -1600,57 +1652,121 @@ <h5><a class="text-decoration-none text-dark " href="#">
...
+        </div>
+
+        <section class="section__newsletter d-none d-md-none d-lg-block">
+            <div class="row">
+                <div class="col-lg-7">
+                    <div class="section__subscribe">
+                        <div class="section_newsletter-header my-3">
+                            Subscribe
+                        </div>
+                        <div style="height: 300px; background: rgb(226, 226, 226);" class="d-flex flex-column flex-md-row
```
Comment (umairarshadbutt) [inline index.html]: Remove inline-style

File: index.html [hunk 5]
```diff
@@ -1600,57 +1652,121 @@ <h5><a class="text-decoration-none text-dark " href="#">
...
+                         justify-content-lg-between align-items-lg-start">
+                            <div class="ml-5 mt-5 mr-2">
+                                <img style="width: 150px;" src="images/newsletter-1.jpg" alt="...">
```
Comment (umairarshadbutt) [inline index.html]: Remove inline-style and move it to CSS file

File: index.html [hunk 6]
```diff
@@ -1600,57 +1652,121 @@ <h5><a class="text-decoration-none text-dark " href="#">
...
+                                <img style="width: 150px;" class="news-letter-img" src="images/cover-mag-footer.jpg"
```
Comment (umairarshadbutt) [inline index.html]: Remove inline-style and move it to CSS file

File: index.html [hunk 7]
```diff
@@ -1600,57 +1652,121 @@ <h5><a class="text-decoration-none text-dark " href="#">
...
+                                    alt="...">
+                            </div>
+                            <div class="ml-5 mt-5">
+                                <span class="mb-4">In-depth Stories You Don't See Elsewhere About Topics You Want to
+                                    Read!</span>
+                                <p class="mt-3 mb-4">All Access + Weekly Delivery.
+                                    Choose A Membership That's Perfect for You!</p>
+                                <button style="width: 40%;" class="btn btn-danger mt-5" type="button">Subscribe
```
Comment (umairarshadbutt) [inline index.html]: Remove inline-style and move it to CSS file....from whole page

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @chrystalme  and @sohaibya 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

Kindly work on the suggested changes  👇.
- [ ] Remove inline CSS from HTML file and move it to CSS file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright) (see inline comments)
- [x] The license link from the readme file has **Application error**, fix it
- [x] Fix Linters issue

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by layzrd — APPROVED** [review]
### Status: Approved 🙌 🎇
Hi @chrystalme and @sohaibya, I am [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR. Congratulations 🎊 👏, you did well on this project. 👏👏👏  I appreciate your hard work on replicating newsweek.com.

Now, you can move ahead.

Please don't forget to merge to the master branch. Good luck with the next projects!

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Comment by sohaibya** [comment]
please approve my project as i was the partner of the chrystalme and worked
with him

**Pull request merged**